src/components/Comments.jsx

```diff
@@ -0,0 +1,42 @@
+import { useState, useEffect } from "react";
+import { getCommentsById } from "../../api";
+
+export default function Comments({id}) {
+    const [comments, setComments] = useState([]);
+    const [errorMsg, setErrorMsg] = useState(null)
+
+    useEffect(() => {
+        getCommentsById(id)
+        .then((commentData) => {
```

Comment:
Please take a look at how to optimistically render here! At the moment, the comment is only added once the request is complete. We should be giving the user visual feedback right away